src/common/utils/netlify.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Utils for organizing and working with our netlify-cms-based content
+ */
+
+import aboutPageContent from 'cms-content/about/about-page.json';
+
+export enum PageType {
+  ABOUT,
+}
+
+const pageTypeToFileMap = {
+  [PageType.ABOUT]: aboutPageContent,
+};
+
+// Retrives page-specific json file:
+export function getContentFile(
+  pageType: PageType,
+): { [fieldName: string]: any } {
+  return pageTypeToFileMap[pageType];
+}
```

Comment:
maybe we could define the interface for the about page here, so we can use it in the `About.tsx` component? Just an idea, maybe for later

Reply:
going to punt for the time being, but agreed that this would be useful